components/02-elements/form/radio/Radio.vue

```diff
@@ -35,7 +38,15 @@
 
 <script>
   export default {
+    model: {
+      prop: 'checked',
+      event: 'change'
+    },
     props: {
+      checked: {
+        type: [String, Object, Number, Boolean],
```

Comment:
Huh, why do we need that many types?

Reply:
This component can accept any of these types.

Reply:
@jszczech 👀

Reply:
This name is misleading, maybe it should be e.g. 'checkedValue' (i changed it). Values can be different than boolean. 

I checked it in some other library, like bootstrap-vue: 

https://github.com/bootstrap-vue/bootstrap-vue/blob/2e5422aa30cd6197846e66016606d9f3e1455f84/src/components/form-radio/form-radio.js#L29

There also added more types than just boolean.